### DISCUSSION TITLE
When using markdown file with knowledge

### DISCUSSION THREAD

**Discussion by mikelfried** [open-webui/open-webui#6230] [General]
# Bug Report

When uploading a markdown file into Knowledge, the preview appears to strip all markdown syntax, making it difficult to determine if this issue is limited to the preview or if it also affects the backend during chunking or when passing it as context to the model. Additionally, markdown formatting is not displayed properly when the model provides the chunk or file as a citation.

## Installation Method

docker

## Environment

- **Open WebUI Version:** [e.g., v0.3.32]

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]

**Confirmation:**

- [V] I have read and followed all the instructions provided in the README.md.
- [V] I am on the latest version of both Open WebUI and Ollama.
- [V] I have included the browser console logs.
- [V] I have included the Docker container logs.
- [V] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

uploading markdown into a knowledge base -> nice preview of it as a markdown -> chunking strategy as a markdown -> nice formatted text citations when using as rag.

## Actual Behavior:

the markdown preview in the knowledge and citations, looked striped from all syntax, without any style.

**Comment by flefevre** [comment]
Same behavior when copy/paste markdown in the Knowledge [add text content] which do not interpretate the markdown format

**Comment by jeffsnack** [comment]
Same here.

I want to add some link like YouTube and Twitch in Workspace,and it will be like

In **Visual Studio Code**:

<img width="735" alt="截圖 2025-02-05 下午2 31 29" src="https://github.com/user-attachments/assets/cf4e0a32-4644-4f47-9f76-d81c2735ecb2" />

In **Open WebUI**:

<img width="933" alt="截圖 2025-02-05 下午2 35 39" src="https://github.com/user-attachments/assets/1a9b9516-7818-45da-b09c-317d0f20881a" />

Maybe we have to wait new feature on next version?

**Comment by SchneiderSam** [comment]
Related: https://github.com/open-webui/open-webui/discussions/16069
